cli/src/pcluster/validators/ec2_validators.py

```diff
@@ -585,6 +585,20 @@ def _validate_with_subnets(
             )
 
 
+class PlacementGroupCapacityTypeValidator(Validator):
+    """Validate the placement group is compatible with the capacity type."""
+
+    def _validate(self, capacity_type: str, placement_group_enabled: bool):
+        if capacity_type == CapacityType.CAPACITY_BLOCK and placement_group_enabled:
+            self._add_failure(
+                "When using a capacity block reservation, a placement group constraint should not be set "
+                "as insufficient capacity errors may occur due to placement constraints outside of the "
+                "reservation even if the capacity reservation has remaining capacity. "
+                "Please remove the placement group for the compute resource.",
+                FailureLevel.ERROR,
```

Comment:
Shall we make it as warning as has been done here? PlacementGrouPlacementGrouPlacementGrouPlacementGrouPlacementGrouPlacementGrouPlacementGrou

Reply:
Good point, iterated.